content/cli/users.md

```diff
@@ -127,18 +124,23 @@ New role `admin` set
 #### Usage 
 ` cfy users delete [OPTIONS] USERNAME`
 
-Delete a user from Cloudify Manager, including from any groups to which they have been assigned.
+Delete a user from Cloudify Manager, 
```

Comment:
**Combine the lines like this:**
Delete a user from Cloudify Manager. You can delete a user only if the user is:

* Not assigned to any tenants
* Not a member of any user groups
* Not the creator of any Cloudify resources (Blueprint, Deployment, Plugin, Secret) on the Manager.